app/src/main/res/values/strings.xml

```diff
@@ -66,5 +66,10 @@
     <string name="data_updated">Data updated!</string>
     <string name="saving_failed">Saving failed!</string>
     <string name="purchase_succeed">Purchase succeed!</string>
+    <string name="enter_the_location_which_weather_you_want_to_observe">Enter the location which weather you want to observe</string>
```

Comment:
@SergeiMikhailovskii try to find short meaningful key for this resource. The one you use now won't make any sense if the resource sightly changes. Resource key must be named after `what the resource is for` or `what it represents`(eg.  `forecast_description`, `location_hint`, etc.). I see that now you use just snake-case form of the english form of the localized resource, which is not how it should be in most cases.